app/views/account_mailer/account_activation.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>VirtualTable</h1>
+
+<p>Hi <%= @user.name %>,</p>
+
+<p>
+Welcome to VirtualTable! Click on the link below to activate your account:
+</p>
+
+<%= link_to "Activate", edit_account_activation_url(@user.activation_token,
+                                                    email: @user.email) %>
```

Comment:
To add I18n here, you probably want to use multiple strings. Something like this:

```erb
<h1><%= t '.title' %></h1>
<%= t '.welcome_html', user: @user.name %>
<%= link_to t('.activate'), edit_account_activation_url(@user.activation_token, email: @user.email) %>
```